src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -330,5 +330,13 @@
     <bean id="batchFileLookupable" parent="batchFileLookupable-parentBean" scope="prototype"/>
     <bean id="batchFileLookupable-parentBean" class="org.kuali.kfs.kns.lookup.KualiLookupableImpl" abstract="true"
           p:lookupableHelperService-ref="batchFileLookupableHelperService"/>
+          
+	<bean id="simpleChartValuesFinder" class="edu.cornell.kfs.coa.businessobject.options.CuSimpleChartValuesFinder"
+		p:parameterService-ref="parameterService"
+		p:keyValuesService-ref="keyValuesService"
+		p:financialSystemUserService-ref="financialSystemUserService"/>
+	
+	<!-- Temporary alias to allow legacy Struts pages to work properly, due to class change in bean override above. -->
+	<alias name="simpleChartValuesFinder" alias="cuSimpleChartValuesFinder"/>
```

Comment:
Was it because of the "financialSystemUserService" reference that you had to move these beans into "cu-spring-sys" instead? It would be preferable to keep these beans in the "KfsBaseAttributeDefinitions.xml" file instead.

Not sure if you tried this yet, but you could try using a service-importing factory bean to import the "financialSystemUserService" bean into the DD bean area. That way, you can reference the service and still keep the values finders within the DD bean files. (You can also try this for the IWantDocumentService injection.)

Many of KFS's "lookup-beans" files import the "spring-kfs-imported-rice-beans.xml" file, which contains an abstract "grlBeanImporter" bean that can be used as the basis for importing KFS services into the DD bean area. You may be able to create child beans of "grlBeanImporter" that can import the FinancialSystemUserService and IWantDocumentService. (You'll have to experiment to see if "spring-kfs-imported-rice-beans.xml" has to be explicitly imported beforehand.)

Reply:
Yes, I moved simpleChartValuesFinder to get access to financialSystemUserService . I will give the import idea a try